File: lua/teste_cnpj.lua
#!/usr/bin/env lua

local cnpj = require "valida_cnpj";

if (arg[1] == "-d") then
    cnpj.print_usage();
else
    if ( cnpj.valida_cnpj( arg[1] ) ) then
        cnpj.formata( arg[1] );
    else
        print("Falha: cnpj "..arg[1].." é inválido.");
    end
end

File: python/teste_cnpj.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import documentos.cnpj
import sys

__cnpj = sys.argv[1]

if documentos.cnpj.valida(__cnpj):
    print "É válido"
    print documentos.cnpj.formata(__cnpj)
else:
    print "É inválido"
